test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -508,7 +508,7 @@
            jruby-puppet (:jruby-puppet jruby-instance)]
        (try
          (testing "Various data types"
-           (is (= "0.0.0.0" (.getSetting jruby-puppet "bindaddress")))
+           (is (= "puppet" (.getSetting jruby-puppet "user")))
```

Comment:
👍  as long as these tests aren't somehow exercised as a part of pe-puppetserver, where the user is pe-puppet ?